src/internal/components/abstract-switch/index.tsx

```diff
@@ -80,7 +80,8 @@ export default function AbstractSwitch({
         className={styles['label-wrapper']}
         aria-disabled={disabled ? 'true' : undefined}
       >
-        <div className={clsx(styles.control, controlClassName)}>
+        {/*Using span, not div for HTML validity*/}
```

Comment:
The comment is no longer needed, we added the html-validator to cover that, see: https://github.com/cloudscape-design/components/blob/main/src/toggle/__tests__/toggle.test.tsx#L95

Reply:
@YueyingLu done!